frontend/public/components/graphs/area.tsx

```diff
@@ -91,9 +91,9 @@ export const AreaChart: React.FC<AreaChartProps> = ({
   const container = React.useMemo(() => {
     if (multiLine) {
       const legendData = processedData.map((d) => ({
-        childName: d[0].description,
-        name: d[0].description,
-        symbol: d[0].symbol,
+        childName: d[0]?.description,
+        name: d[0]?.description,
+        symbol: d[0]?.symbol,
```

Comment:
Do we know if it's safe to pass `undefined` for these values? How does it impact the chart when this happens?

Reply:
I followed this down through the ChartLegendTooltip component and it looks like it should be okay if these are undefined. I'm not 100% sure what effect this would have on the chart since in this case `d[0] === undefined` means there is an empty data set. Let me dig a little bit more.